GitForWindowsHelper/slash-commands.js

```diff
@@ -141,21 +141,33 @@ module.exports = async (context, req) => {
             )
 
             const triggerWorkflowDispatch = require('./trigger-workflow-dispatch')
-            const answer = await triggerWorkflowDispatch(
-                context,
-                await getToken(),
-                'git-for-windows',
-                'git-for-windows-automation',
-                'build-and-deploy.yml',
-                'main', {
-                    package: package_name,
-                    repo,
-                    ref,
-                    actor: commenter
-                }
-            )
+            const triggerBuild = async () =>
+                await triggerWorkflowDispatch(
+                    context,
+                    await getToken(),
+                    'git-for-windows',
+                    'git-for-windows-automation',
+                    'build-and-deploy.yml',
+                    'main', {
+                        package: package_name,
+                        repo,
+                        ref,
+                        actor: commenter
+                    }
+                )
+
             const { appendToIssueComment } = require('./issues')
-            const answer2 = await appendToIssueComment(context, await getToken(), owner, repo, commentId, `The workflow run [was started](${answer.html_url})`)
+            const appendToComment = async (text) =>
+                await appendToIssueComment(
+                    context,
+                    await getToken(),
+                    owner,
+                    repo,
+                    commentId,
+                    text
+                )
+            const answer = await triggerBuild()
+            const answer2 = appendToComment(`The workflow run [was started](${answer.html_url})`)
```

Comment:
Don't we need await here?

Reply:
Absolutely!

Reply:
Fixed, in https://github.com/git-for-windows/gfw-helper-github-app/pull/14/commits/98372b343dd4ec9e5c24501fc83dd380629d71cb:

![image](https://user-images.githubusercontent.com/127790/208875893-eedeb8e7-a044-4bcd-ad25-190f8db9e9a1.png)